webapp/channels/src/utils/utils.tsx

```diff
@@ -1212,6 +1212,9 @@ export function clearFileInput(elm: HTMLInputElement) {
     }
 }
 
+/**
+ * @deprecated Use react-intl instead, only place its usage can be justified is in the redux actions
```

Comment:
why can you use it in redux actions?

Reply:
React Intl relies on the translation functions and locale coming through as part of the React context. Redux actions don't have access to that directly, so they can't use React Intl unless we pass the `intl` object into them.

I've know I've wanted to find a better option for that other than `localizeMessage` or passing `intl` into every action as an argument, but I haven't spent the time to go do it